cmd/migrate/config.go

```diff
@@ -35,4 +35,8 @@ var (
 
 	flagConfigDirectory = pflag.String("config.source", defaultConfigDirectory, "directory of the configuration file")
 	flagConfigFile      = pflag.String("config.file", "", "configuration file name without extension")
+
+	// goto command flags
+	flagDirty   = pflag.Bool("dirty", false, "migration is dirty")
```

Comment:
maybe this should be a verb instead of a noun. for example, 'handle-dirty', 'force-dirty-handling'. i'm open to the naming here but `dirty` by itself is not invocative about what this does.

Reply:
`force-dirty-handling` seems clear and concise on what it is supposedly doing

Reply:
That works for me